src/breadcrumb-group/index.tsx

```diff
@@ -12,7 +12,7 @@ export default function BreadcrumbGroup<T extends BreadcrumbGroupProps.Item = Br
   items = [],
   ...props
 }: BreadcrumbGroupProps<T>) {
-  const baseComponentProps = useBaseComponent('BreadcrumbGroup');
+  const baseComponentProps = useBaseComponent('BreadcrumbGroup', { items });
```

Comment:
Not sure about logging nested objects. What is the use-case on consumer end for it? Are we sure they won't get `[object Object]` logged in their metrics?